[5.3] note a bug that makes completing the section difficult

ziyadedher opened the pull request:
This commit highlights #27 in the tutorial directly and gives users a workaround.

BartMassey commented on mdbook/src/05-meet-your-software/debug-it.md:
```diff
@@ -33,7 +33,7 @@ The `../../..` in this command is needed, since each example project is in a "wo
 chapter in Rust Book] for more.
 
-> **NOTE** If `cargo-embed` prints a lot of warnings here don't worry about it. As of now it does
+> **NOTE**: If `cargo-embed` prints a lot of warnings here don't worry about it. As of now it does
```
The convention in this book is to leave the colon `:` off. I agree that it's a bit janky, but it's everywhere.

ziyadedher replied:
I pattern matched from this file and didn't take a look at the rest, but this file mostly has the with-colon version! Will change, though.

BartMassey commented on mdbook/src/05-meet-your-software/debug-it.md:
````diff
@@ -49,9 +49,19 @@ Remote debugging using :1337
+> Remote debugging using :1337
+> init::__cortex_m_rt_main () at mdbook/src/05-meet-your-software/examples/init.rs:19
+> 19              asm::nop();
+> ```
````
Not seeing the actual `monitor reset halt` in this terminal fragment?

ziyadedher replied:
Added in!

BartMassey reviewed (changes requested):
Thanks much for this. We'll take it if — *and only if* — you pinky-promise to submit another PR to remove it once `probe-rs` is fixed :grinning: .

Please see some notes about needed changes I've stuck in.

ziyadedher commented:
Addressed the feedback and yeah, I'll keep an eye out :P

I might forget, feel free to tag me in and I can PR it back out. If you'd prefer to keep this PR out though, for whatever reason, feel free; I have the info in my brain now haha.

ziyadedher commented:
(im unable to merge this myself btw!)

BartMassey commented:
Yes, waiting for another rust-embedded person to glance at it and merge it. If it doesn't happen soon, I'll merge it myself, but always nice to get an extra set of eyeballs.
